File: 프로그래머스/level1/8주차.cpp
#include <string>
#include <vector>
#include <iostream>
#include <algorithm>

using namespace std;

int solution(vector<vector<int>> sizes) {
    int answer = 0;
    int h = 0;
    int w = 0;
    for(int i=0; i<sizes.size(); i++){
        if(sizes[i][0]<=sizes[i][1]){
            h = max(h, sizes[i][1]);
            w = max(w, sizes[i][0]);
        }
        else{
            w = max(w, sizes[i][1]);
            h = max(h, sizes[i][0]);
        }
    }
    answer = h*w;
    return answer;
}

File: 프로그래머스/level1/8주차.js
function solution(sizes) {
    var answer = 0;
    let h = 0;
    let w = 0;
    for(let i=0; i<sizes.length; i++){
        if(sizes[i][0]<=sizes[i][1]){
            h = Math.max(h, sizes[i][1]);
            w = Math.max(w, sizes[i][0]);
        }
        else{
            w = Math.max(w, sizes[i][1]);
            h = Math.max(h, sizes[i][0]);
        }
    }
    answer = h*w;
    return answer;
}